### ISSUE TITLE
Timeout error after upgrade

### ISSUE THREAD

**Issue by loganmarchione** [goodwithtech/dockle-action#5]
I recently upgraded the GitHub Action from `goodwithtech/dockle-action@0.1.0` to `goodwithtech/dockle-action@v0.1.1`, but am now getting this error in all of my GitHub Actions runs.

```
 2023/09/10 05:06:00 could not parse  as duration for flag timeout, t: time: invalid duration ""
```

Here is how I'm calling the Action.

```
    - name: Container scan with Dockle
      uses: goodwithtech/dockle-action@v0.1.1
      with:
        image: '${{ env.USER }}/${{ env.REPO }}:${{ env.VERSION }}'
        format: 'list'
        exit-code: '1'
        exit-level: 'warn'
        ignore: 'CIS-DI-0001'
```

**Comment by tomoyamachi** [comment]
@loganmarchione Unfortunately, I did not release v0.1.1.
https://github.com/goodwithtech/dockle-action/releases

I have now released version 0.1.2.
Please try using the 0.1.2 version.

**Comment by loganmarchione** [comment]
I saw there was a [tag](https://github.com/goodwithtech/dockle-action/tags) for `v0.1.1` which is what [renovatebot picked up on](https://github.com/loganmarchione/docker-webdav-nginx/pull/17).

Just upgraded to `v0.1.2` and it's working, thanks for the quick response! 🙏
